CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# Contributing to the Glean Dictionary
+
+Thank you for your interest in contributing to the Glean Dictionary. This
+document tries to codify some best practices for contribution to this
+repository.
+
+## Participation guidelines
+
+All communication (on the Glean Dictionary channel, on mailing lists, and in
+pull requests and issues) is expected to follow the
+[Mozilla Community Participation Guidelines](https://www.mozilla.org/about/governance/policies/participation/).
+For more information, see the [code of conduct document](./CODE_OF_CONDUCT.md)
+in the root of this repository.
+
+## Filing issues
+
+File an issue if you have a bug report or feature request that you (personally)
+do not intend to work on right away _or_ you would like additional feedback on
+your approach before starting implementation work. If you found a bug (or small
+missing feature) and you want to start implementing it immediately (or already
+have a solution), go ahead and skip straight to making a pull request (see
+below).
+
+To help with triage, issues should have a descriptive title. Examples of good
+issue titles:
+
+- Storybook doesn't render components with styling
+- Add a filter box to the applications page
+
+In the issue itself, provide as much information as necessary to help someone
+reading it understand the nature of the problem (and provide feedback). For
+examples of this, look at some of the
+[fixed issues](https://github.com/mozilla/glean-dictionary/issues?q=is%3Aissue+is%3Aclosed)
+filed by the project maintainers.
+
+## Opening pull requests
+
+Like issues, pull requests should have a descriptive title to help with triage.
+However there are two things that are different:
+
+- Instead of pointing out a problem, they should describe the solution
+- If a pull request fixes a specific issue, the title should specify
+  `(fixes #X)` (where X refers to the issue number)
```

Comment:
Github will cross reference the issue if this is put into the description of the PR.

Reply:
True, but if it's in the title commit messages will also get the information.